DuckDuckGo/PrivacyDashboard/View/PrivacyDashboardViewController.swift

```diff
@@ -283,7 +284,7 @@ extension PrivacyDashboardViewController: PrivacyDashboardControllerDelegate {
                 let report = try await makeBrokenSiteReport(source: source)
                 try toggleProtectionsOffReporter.report(report, reportMode: .toggle)
             } catch {
-                os_log("Failed to generate or send the broken site report: %@", type: .error, error.localizedDescription)
+                Logger.general.error("Failed to generate or send the broken site report: \(error.localizedDescription, privacy: .public)")
```

Comment:
should it be public?

Reply:
a lot of strings should be public but I just left what was there before, this can be fixed by the owner if needed